cmd/end2endtest/overwrite.go

```diff
@@ -75,19 +75,17 @@ func (t *E2EOverwriteElection) Run() error {
 
 	// overwrite the previous vote (choice 0) associated with account of index 0, using enough time to do it in the nextBlock
 	// try to make 3 overwrites (number of choices passed to the method). The last overwrite should fail due the maxVoteOverwrite constrain
-	err := t.overwriteVote([]int{0, 1, 0}, 0, nextBlock)
+	err := t.overwriteVote([]int{0, 1, 2}, 0, nextBlock)
```

Comment:
In `newTestElectionDescription` we have declared only 2 possible choices allowed (`0` and `1`), so I think this value of `2` will not be taken into account

Reply:
that's so true! good catch <3

i refactored newTestElectionDescription to solve this